packages/editor/src/plugins/fill-in-the-blanks-exercise/renderer.tsx

```diff
@@ -137,6 +153,14 @@ export function FillInTheBlanksRenderer(props: FillInTheBlanksRendererProps) {
           </BlankDraggableArea>
         ) : null}
 
+        {mode === 'drag-and-drop' && isEditing ? (
+          <BlankDraggableDummyArea
+            extraDraggableAnswers={
+              extraDraggableAnswers as FillInTheBlanksExerciseProps['state']['extraDraggableAnswers']
+            }
+          />
+        ) : null}
+
```

Comment:
Hm, here the renderer component is doing stuff that feels like it should instead be done by the editor component. Can we move this into the `editor.tsx`?

Reply:
Could you please elaborate?

Reply:
As far as I understood it, the idea behind the split into `editor.tsx`, `renderer.tsx` and `static.tsx` is that ...
- `editor.tsx` renders and provides all editor functionality
- `static.tsx` handles the static rendering view
- `renderer.tsx` renders the plugin based on its state and is used by both `editor.tsx` and `static.tsx`.
So, using that logic anything that should only be available in the editor should live in `editor.tsx`.

Reply:
Ah, good point!